lib/article-tile/index.js

```diff
@@ -3,11 +3,13 @@ import React, { Component, PropTypes } from 'react';
 require('./style.css');
 
 export class ArticleTile extends Component {
-  shouldComponentUpdate (nextProps) {
-    if (nextProps.tile.name === this.props.tile.name) {
-      return false;
-    }
-  }
+  // shouldComponentUpdate (nextProps) {
```

Comment:
prefer code to be removed instead of commented out unless you are testing something in which case leave an explanatory comment above the commented out code...